IGSIOCalibration/vtkIGSIOSpinCalibrationAlgo.h

```diff
@@ -0,0 +1,64 @@
+/*=IGSIO=header=begin======================================================
+Program: IGSIO
+Copyright (c) Laboratory for Percutaneous Surgery. All rights reserved.
+See License.txt for details.
+=========================================================IGSIO=header=end*/
+
+#ifndef __vtkIGSIOSpinCalibrationAlgo_h
+#define __vtkIGSIOSpinCalibrationAlgo_h
+
+// Local includes
+#include "igsioConfigure.h"
+#include "igsioCommon.h"
+#include "vtkIGSIOAbstractCalibrationAlgo.h"
+#include "vtkIGSIOCalibrationExport.h"
+
+// VTK includes
+#include <vtkCommand.h>
+#include <vtkObject.h>
+#include <vtkMatrix4x4.h>
+
+class vtkIGSIOTransformRepository;
+class vtkXMLDataElement;
+
+//-----------------------------------------------------------------------------
+
+/*!
+  \class vtkIGSIOSpinCalibrationAlgo
+  \brief Spin calibration algorithm to calibrate a stylus. It determines the direction of the stylus shaft.
```

Comment:
it would be nice to write a bit about how the algorithm works and how it is related to pivot calibration